sdk/core/azure-core/src/main/java/com/azure/android/core/util/logging/AndroidClientLogger.java

```diff
@@ -61,48 +63,80 @@ public void setLogLevel(@LogLevel int logLevel) {
         this.logLevel = logLevel;
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @Override
     public void debug(String message) {
         if (LOG_LEVEL_DEBUG >= this.logLevel) {
             Log.d(tag, message);
         }
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @Override
     public void debug(String message, Throwable throwable) {
         if (LOG_LEVEL_DEBUG >= this.logLevel) {
             Log.d(tag, message, throwable);
         }
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @Override
     public void info(String message) {
         if (LOG_LEVEL_INFO >= this.logLevel) {
             Log.i(tag, message);
         }
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @Override
     public void info(String message, Throwable throwable) {
         if (LOG_LEVEL_INFO >= this.logLevel) {
             Log.i(tag, message, throwable);
         }
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @Override
     public void warning(String message) {
         if (LOG_LEVEL_WARNING>= this.logLevel) {
             Log.w(tag, message);
         }
     }
 
+    /**
+     * {@inheritDoc}
+     */
```

Comment:
nit; given this is internal type we can remove inherit-javadoc for all overriden methods